### PR TITLE
Draggable Label

### PR DESCRIPTION
**Pull request by rmenegaux**
Adding a drag behavior for `Label`.
 
The `Label` coordinates `x` and `y` are updated accordingly at the end of the drag.

This also cleans up the `Label.js` code.

### CODE REVIEW

**Comment by dmadeka** [comment]
Looks fine to me, but I guess it would be better to move to a generic `Objects` mark and allow `Scatter` and `Label` to be instances of it.

**Comment by rmenegaux** [comment]
@dmadeka agreed
This works for now though, pending a move to a more generic `Scatter`

**Comment by dmadeka** [comment]
Can you add `enable_move` to `Wealth of Nations` and we should be good

**Comment by dmadeka** [comment]
Also, the `mouseover` icon when you drag is unique to the `Label` now

**Comment by rmenegaux** [comment]
@dmadeka good to go

File: js/src/Label.js [hunk 1]
```diff
@@ -65,79 +64,95 @@ define(["d3", "./Mark"], function(d3, MarkViewModule) {
...
                 this.el.select(".label")
-                    .style("fill", this.color);
+                    .style("fill", color);
             }
         },
+        set_drag_behavior: function() {
+            var label = this.el.select(".label");
+            if (this.model.get("enable_move")) {
+                label.call(this.drag_listener)
+                    .style("cursor", "move");
```
Comment (dmadeka) [inline js/src/Label.js]: im not sure about this. no other `drag` functionality has a cursor. Typically, we make the element larger, but if we move all drag to this I'm okay with this change. Let's keep it for another PR.

File: examples/Wealth of Nations.ipynb [hunk 2]
```diff
@@ -131,7 +131,7 @@
    "source": [
     "#### Creating the Tooltip to display the required fields\n",
     "\n",
-    "`bqplot`s native `Tooltip` allows us to simply display the data fields we require on a mouse-interaction."
+    "`bqplot`'s native `Tooltip` allows us to simply display the data fields we require on a mouse-interaction."
```
Comment (dmadeka) [inline examples/Wealth of Nations.ipynb]: 's means `bqplot` is

**Comment by rmenegaux** [comment]
@dmadeka removed the 'move' cursor for now
